octis/models/ETM.py

```diff
@@ -16,7 +16,7 @@ class ETM(BaseETM):
     def __init__(self, num_topics=10, num_epochs=100, t_hidden_size=800, rho_size=300, embedding_size=300,
                  activation='relu', dropout=0.5, lr=0.005, optimizer='adam', batch_size=128, clip=0.0,
                  wdecay=1.2e-6, bow_norm=1, device='cpu', top_word=10, train_embeddings=True, embeddings_path=None,
-                 use_partitions=True):
+                 embeddings_type='pickle', binary_embeddings=True, use_partitions=True):
```

Comment:
Can you add the comments to explain the parameters "embeddings_type" and "binary_embeddings"? Including the type and default values (I'll take care of the others parameters).

Reply:
Done